operators/community-windows-machine-config-operator/5.1.2/manifests/community-windows-machine-config-operator.5.1.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,349 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: "false"
+    createdAt: "2022-07-06T09:05:36Z"
+    description: An operator that enables Windows container workloads on OCP
+    olm.skipRange: '>=4.0.0 <5.1.1'
```

Comment:
since 5.1.1 is a working version, we dont want to be skipping anything. we should just be upgrading (replace) from 5.1.1